Teaching/Stanford_CS344_2018/simple_router.config.template

```diff
@@ -0,0 +1,12 @@
+set_crc16_parameters calc_2 0x1021 0xffff 0x0000 false false
```

Comment:
I'm not too familiar with p4app, is there a reason for the `.template` suffix in the file name?

Reply:
p4app needs simple_router.config which generated by setup.py by appending a set of rules to the contents of simple_router.config.template.